deploy/kubernetes/rbac.yaml

```diff
@@ -85,9 +85,6 @@ metadata:
 rules:
 # Only one of the following rules for endpoints or leases is required based on
 # what is set for `--leader-election-type`. Endpoints are deprecated in favor of Leases.
-- apiGroups: [""]
-  resources: ["endpoints"]
-  verbs: ["get", "watch", "list", "delete", "update", "create"]
```

Comment:
This is the same change that was made earlier in https://github.com/kubernetes-csi/external-resizer/commit/15a912cfe617c0d38a84cd2b8639c2013b912358

/lgtm